prime-router/src/main/kotlin/azure/LookupTableFunctions.kt

```diff
@@ -87,21 +82,61 @@ class LookupTableFunctions(
         @BindingName("tableName") tableName: String,
         @BindingName("tableVersion") tableVersion: Int
     ): HttpResponseMessage {
-        return getOktaAuthenticator().checkAccess(request) {
-            try {
-                if (!lookupTableAccess.doesTableExist(tableName, tableVersion))
-                    HttpUtilities.notFoundResponse(
-                        request,
-                        "Table $tableName with version $tableVersion does not exist."
-                    )
-                else {
-                    val rows = lookupTableAccess.fetchTable(tableName, tableVersion)
-                    HttpUtilities.okResponse(request, convertTableDataToJsonString(rows))
-                }
-            } catch (e: Exception) {
-                logger.error("Unable to fetch lookup table with version $tableVersion", e)
-                HttpUtilities.internalErrorResponse(request)
+        // Do authentication
+        val claims = AuthenticatedClaims.authenticate(request)
+            ?: return HttpUtilities.unauthorizedResponse(request, authenticationFailure)
+        // No authorization check!   Anyone authenticated in ReportStream is allowed to access this function.
+        logger.info("User ${claims.userName} is authorized for endpoint ${request.uri}")
+
+        return try {
+            if (!lookupTableAccess.doesTableExist(tableName, tableVersion))
+                HttpUtilities.notFoundResponse(
+                    request,
+                    "Table $tableName with version $tableVersion does not exist."
+                )
+            else {
+                val rows = lookupTableAccess.fetchTable(tableName, tableVersion)
+                HttpUtilities.okResponse(request, convertTableDataToJsonString(rows))
+            }
+        } catch (e: Exception) {
+            logger.error("Unable to fetch lookup table with version $tableVersion", e)
+            HttpUtilities.internalErrorResponse(request)
+        }
+    }
+
+    /**
+     * Fetch the data for a specific table version.
+     */
+    @FunctionName("getActiveLookupTableData")
+    fun getActiveLookupTableData(
+        @HttpTrigger(
+            name = "getActiveLookupTableData",
+            methods = [HttpMethod.GET],
+            authLevel = AuthorizationLevel.ANONYMOUS,
+            route = "lookuptables/{tableName}/content"
+        ) request: HttpRequestMessage<String?>,
+        @BindingName("tableName") tableName: String,
+    ): HttpResponseMessage {
+        // Do authentication
+        val claims = AuthenticatedClaims.authenticate(request)
+            ?: return HttpUtilities.unauthorizedResponse(request, authenticationFailure)
+        // No authorization check!   Anyone authenticated in ReportStream is allowed to access this function.
+        logger.info("User ${claims.userName} is authorized for endpoint ${request.uri}")
```

Comment:
Remove or change to debug

Reply:
Keeping auth related logging as 'info' so it remains in the logs.